algorithms/CSharp/src/Maths/fibonacci-series.cs

```diff
@@ -0,0 +1,29 @@
+using System;
```

Comment:
Add a new line before open the curly bracket and please add the `Main()` section due to all C# algorithm should work with [.Net Fiddle](https://dotnetfiddle.net).

Reply:
I'm pretty new in everything related to git and working on sharing projects, so accept my apologies for my mistakes. 
Is it necessary to have a Main inside the class? I've seen the other two classes made for other users,  and I think they should delete the main method inside the class and use the test class to test it. We could maybe create just one main Main to call all the methods.

Reply:
Well... this the role when the C# project created, is for the user don't have to download the repo and run it with SDK or runtime to analyze the algorithm.

Reply:
This issue is not solved, please add the `Main` method and try using .Net Fiddle to test it